lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -1213,10 +1213,6 @@ bool TypeLoweringVisitor::visitDecl(InstanceOp op) {
   if (skip) {
     return false;
   }
-  if (!sym || sym.getValue().empty())
```

Comment:
(Is this code not needed?)

Reply:
This wasn't supposed to make it in the PR 😂 I was just curious what would break when I removed it, and then opened this issue to ask about it over here https://github.com/llvm/circt/issues/3505